vars/pipelineJava8.groovy

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env groovy
+
+// Java Backend Reference Jenkinsfile
+def call(
+  environment,
+  gitCredentialsId,
+  sonarqubeCredentialsId,
+  argocdCredentialsId,
+  artifactRepoCredentialsId
+) {
+
+  String JENKINS_WORKER_IMAGE_JNLP    = 'quay.io/tssc/tssc-ci-agent-jenkins:latest'
+  String JENKINS_WORKER_IMAGE_MAVEN   = 'quay.io/tssc/tssc-tool-maven:latest'
+  String JENKINS_WORKER_IMAGE_BUILDAH = 'quay.io/tssc/tssc-tool-buildah:latest'
+  String JENKINS_WORKER_IMAGE_ARGOCD  = 'quay.io/tssc/tssc-tool-argocd:latest'
+  String JENKINS_WORKER_IMAGE_SKOPEO  = 'quay.io/tssc/tssc-tool-skopeo:latest'
+  String JENKINS_WORKER_IMAGE_SONAR   = 'quay.io/tssc/tssc-tool-sonar:latest'
+  String REGISTRY_SECRET_NAME         = 'quay-basic-auth'
+
+  pipeline {
+
+    agent {
+      kubernetes {
+        label "jenkins-tssc-reference-app-quarkus-rest-json-${env.BUILD_ID}"
```

Comment:
What do you think of parameterizing the application name to be used here and throughout, if necessary?

Reply:
fixed